modules/GraphvizLayout/src/main/resources/org/icculus/chunky/gephigraphviz/Bundle.properties

```diff
@@ -0,0 +1,45 @@
+# To change this template, choose Tools | Templates
+# and open the template in the editor.
+OpenIDE-Module-Display-Category=Layout
+OpenIDE-Module-Long-Description=\
+    A simple plugin that calls graphviz at the command line and uses it to lay out a graph. \
+    Graphviz has a selection of very high-quality layout algorithms available.
+OpenIDE-Module-Name=GraphvizLayout
```

Comment:
This is the "branding" name of your plugin, so I would rather recommend "Graphviz Layout"